### DISCUSSION TITLE
Admin login question

### DISCUSSION THREAD

**Discussion by lalaluk** [drakkan/sftpgo#1443] [Q&A] [answered]
I have two questions:

1. Can admin login and see files in FTP, or is admin only used for managing users and related settings?

2. If I want an admin to be able to see the ftp files of certain users, is it possible to do that?

**Comment by drakkan** [comment, marked as answer]
> I have two questions:
> 
> 1. Can admin login and see files in FTP, or is admin only used for managing users and related settings?

admins can manage users and other settings, they cannot login via SFTP, FTP, WebDAV

> 2. If I want an admin to be able to see the ftp files of certain users, is it possible to do that?

This is not possible, you must create a user whose home directory contains the home directory of all other users

**Reply by lalaluk** [reply]
Thank you I create a user whose home directory contains the home directory of all other users
It solve my problem perfectly
